### ISSUE TITLE
List watched apps

### ISSUE THREAD

**Issue by md5** [errbit/errbit#987]
As far as I can tell, there isn't currently a place for a user to see a list of their watched apps. I think this would be a nice feature to have.

**Comment by stevecrozz** [comment]
I agree.

**Comment by md5** [comment]
@stevecrozz Do you think that just adding a "Watched?" column to the main "Apps" view would make sense, or should a separate `/apps/watched` view be created?

I could also see it making sense to have "Watch"/"Unwatch" buttons on each row of the main "Apps" view.

**Comment by stevecrozz** [comment]
We could have some filters that pass query string arguments to the /apps index. Like /apps?watched_by=me. There could be filter links at the top of the index to allow the user to specify these filters. I really enjoy the way github does it for issues:
![issues errbit errbit](https://cloud.githubusercontent.com/assets/133328/10858908/b4b2fcd4-7f15-11e5-8231-b8b8981f077b.png)

This is far more power than what we need right now, but I think it would be nice to head in this general direction in terms of how our index views can be filtered.

To start with, if we just had "All Apps" and "Watched by me" links at the top in the same way github has "open" and "closed" links for filtering the issues list, I think that would be awesome.

**Comment by md5** [comment]
I had a bit of time this evening to look at this and started a branch: https://github.com/appropriate/errbit/tree/watched-apps

**Comment by mfrederickson** [comment]
@stevecrozz what is the status on this?  is this in the 7.0 milestone too?

**Comment by stevecrozz** [comment]
@mfrederickson I think it could be depending on when it is finished. @md5's branch looks pretty good. I might want to changed the text 'watched_by' to 'watcher' to give it a simpler appearance. Other than that, we need a spec or two and it should be mergable. Thanks for the reminder.

**Comment by md5** [comment]
@stevecrozz Are you talking about the URL parameter name or something else?

**Comment by stevecrozz** [comment]
Yeah. I'm thinking someday this will be part of a larger advanced search DSL that people can type into the search bar and I think watcher may be slightly easier to remember/type than watched_by.
